tasks/configure-RedHat.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+# XXX command flags for nginx is hard-coded in
+# /usr/lib/systemd/system/nginx.service
```

Comment:
this comment should be in README.md.
reading README.md, i expected it would be implemented in the future.